charts/sentry-apps/Chart.yaml

```diff
@@ -14,4 +14,4 @@ maintainers:
 dependencies:
   - name: argoconfig
     repository: https://charts.adfinis.com
-    version: ~0.6.1
+    version: 0.7.4
```

Comment:
i'm kinda not sure if this one doesn't introduce any bc breakage, will have to take a closer look

Reply:
I'm not sure if sentry should still be a priority for us. I'm pushing our developers towards the cloud hosted solution right now.

Reply:
we could keep this as is, und upgrade the  rest of the chart deps. 

Btw is it a patchfix bump or minor bump?